_summaries/sitzmann2020implicit_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: sitzmann2020implicit
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Overview
+The authors introduce sinusoidal representation networks (SIRENS) as an improvement over current network architectures (namely ReLU MLPs) for implicit neural representations of signals. They perform experiments on a range of signals (image, video, sound) as well as on a number of equations rooted in physics applications. The main benefit of SIRENS is the ability to more easily compute their derivatives, as well as any higher order derivatives required.
+
+## Background
+Implicit neural representations serve as a method for continuous encoding of signals via neural networks. These representations provide benefits over traditional discrete representations since they can be more memory efficient and are differentiable.
+
+### ReLU MLPs
+The authors note that most recent work in implicit neural representations have utilized the ReLU MLP architecture. This architecture is limited in the fact that it lacks the capacity to model details well and its incapable of modeling information in higher-order derivatives (since its second derivative is 0).
+
+## SIREN
+The authors propose SIREN as a simple architecture that utilizes sine as a periodic activation function. It is formulated as shown below:
+
+![SIREN Formulation](sitzmann2020implicit_2a.png)
+
+where each phi term represents a layer of the network. SIRENs are shown to converge faster than baseline architectures and have some interesting properties; for example the derivative of a SIREN is also a SIREN (given that the derivative of sine is cosine, which in itself is a phase-shifted sine).
+
+### Basic Image Usecase
+The authors provide an example of trying to parameterize a discrete image. 
+In this case each image defines a dataset D = {(x<sub>i</sub>, f(x<sub>i</sub>))}<sub>i</sub> where x<sub>i</sub> is the pixel coordinate (x<sub>i</sub>, y<sub>i</sub>) and f(x<sub>i</sub>)) is the associated RGB color. Network phi (trying to predict f(x<sub>i</sub>)) is fit using an L2 loss function and trained over the dataset.
```

Comment:
For this image usecase section, it maybe a good idea to add the example of learning from space which i think was the only experiment they trained a network + showing the pixel-wise MSE table.